### ISSUE TITLE
0.3.0 version download

### ISSUE THREAD

**Issue by Irisapo** [broadinstitute/CellBender#159]
Hey, 

Want to use Cellbender 0.3 and so want to check if the master branch code is up to date to the latest preprint version 0.3.0
It's said to be 0.2.0 in the setup.py of this master branch. 
I also saw there is an archived branch of CellBender 0.3.0. 
Any suggestion on which one to use? 

Thanks, 
Shiyi

**Comment by sjfleming** [comment]
Hi Shiyi,

Yes I am a little slow in merging the new 0.3.0 changes into the master branch.  The master branch is currently 0.2.1.

The version of the code that was used to generate figures in the preprint is this commit
```
git checkout be71f8fb4ff941f18ed6259147184cd48cd9d727
```

Within the next few weeks, hopefully (but certainly before the paper gets published), I will be releasing version 0.3.0 officially.  I'm still making some improvements.

The very latest branch where I'm currently doing active development is `sf_dev_0.3.0_postreg`, but I wouldn't really recommend using that branch, since I am actively changing stuff and refactoring a little bit.

So, the above commit might be what you want to use for now.

There is a also a docker image currently living at
```
us.gcr.io/broad-dsde-methods/cellbender:dev
```
which is currently something close to v0.3.0.  (The problem is that "0.3.0" is a moving target.)  The disclaimer about the above docker image is that the "dev" tag is subject to change in the future.  If you want a permanent link to that image, it can be accessed using the following
```
docker pull us.gcr.io/broad-dsde-methods/cellbender@sha256:f68433cab76cbe6fc7e3b8c9d7546c7cc4aa3801a68d7f570daf68a8f5dbbd8d
```

**Comment by Irisapo** [comment]
Hi Stephen, 

Thank you for pointing me to these 0.3 codebases. Appreciate it. 
Look forward to the official 0.3.0 release and the publication!

Best, 
Shiyi
